pkg/environment/environment.go

```diff
@@ -182,10 +225,20 @@ func LogEnvVariables(log *zerolog.Logger) {
 	envVarArray := [...]string{CatalogConnectorServiceAddressKey, VaultAddressKey, VaultModulesRoleKey,
 		EnableWebhooksKey, ConnectionTimeoutKey, MainPolicyManagerConnectorURLKey,
 		MainPolicyManagerNameKey, LoggingVerbosityKey, PrettyLoggingKey, DatapathLimitKey,
-		CatalogConnectorServiceAddressKey, DataDir, ModuleNamespace, ControllerNamespace, ApplicationNamespace}
+		CatalogConnectorServiceAddressKey, DataDir, ModuleNamespace, ControllerNamespace, ApplicationNamespace,
+		ResourcesPollingInterval, HelmWaitTimeout}
 
 	log.Info().Msg("Manager configured with the following environment variables:")
 	for _, envVar := range envVarArray {
 		logEnvVariable(log, envVar)
 	}
+	interval, err := GetResourcesPollingInterval()
+	if err != nil {
+		log.Error().Msg("error getting " + ResourcesPollingInterval + ". Setting the default to " +
+			interval.String())
+	}
+	timeout, err := GetHelmWaitTimeout()
+	if err != nil {
+		log.Error().Msg("error getting" + HelmWaitTimeout + " Setting the default to " + timeout.String())
+	}
```

Comment:
get the new variables out of the loop and handle them separately. Add "else" to if err != nil to print the values.

Reply:
@shlomitk1 I find that this can be a little confusing - if the env value is empty then we will print the default value but there is no way to tell that after we end the call to `GetResourcesPollingInterval` or `GetHelmWaitTimeout`. also, the printed value is not the value of the env var but rather of the returned value of  `GetResourcesPollingInterval` or `GetHelmWaitTimeout`.
another option is to initialize a new var based on the env var , log it then, and return that value afterward.  what do you think?

Reply:
Don't use these variables in the logEnvVar loop. Add else statement to print the value, like:
        timeout, err := GetHelmWaitTimeout()
	if err != nil {
		log.Warning().Msg("error getting" + HelmWaitTimeout + " Setting the default to " + timeout.String())
	} else {
	        log.Info().Msgf("%s set to \"%s\"", HelmWaitTimeout, timeout.String())
	}

Reply:
OK, I change the code. Thanks